ansible/inventories/group_vars/all

```diff
@@ -32,11 +32,6 @@ logstash_forwarder:
 # environment
 parasitic: false
 
-denyhosts_allowed:
```

Comment:
I may have jumped the gun with this comment.
I see that even though this pull request is closed, the ticket for it is still in progress :construction: :construction_worker: